staff/staff_views_helper.py

```diff
@@ -189,6 +204,40 @@ def email_announcement(request, form, email_list):
     messages.add_message(request, messages.SUCCESS, "Recipients Successfully Emailed")
 
 
+def email_posted_form(request, form, email_list):
+    subject = form.cleaned_data.get("name")
+    print()
```

Comment:
We can probably get rid of these print() calls

Reply:
Taken care of